client/lib/signup/step-actions/index.js

```diff
@@ -1270,7 +1269,7 @@ export function isNewOrExistingSiteFulfilled( stepName, defaultDependencies, nex
 	}
 }
 
-export const buildUpgradeFunction = ( planProps, cartItem ) => {
+export const buildUpgradeFunction = ( planProps, cartItem, addOnItem ) => {
```

Comment:
**Note:**

Amendments to this file allow us to add storage add-ons to the shopping cart in **signup** onboarding flows